src/libstd/sys/unix/backtrace/printer.rs

```diff
@@ -0,0 +1,246 @@
+// Copyright 2014-2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+pub use self::dladdr::*;
+pub use self::libbacktrace::*;
+
+#[cfg(not(any(target_os = "macos", target_os = "ios")))]
+mod dladdr { }
+#[cfg(any(target_os = "macos", target_os = "ios"))]
+mod dladdr {
```

Comment:
Can this follow the same convention as above for generating backtraces? Basically something like:

```rust
// mod.rs
#[cfg(...)] #[path = "dladdr.rs"] mod print;
#[cfg(...)] #[path = "libbacktrace.rs"] mod print;

fn foo() {
    print::print(...)
}
```